changelog.d/20230307_164111_austin_add_metalad_extractor.md

```diff
@@ -0,0 +1,5 @@
+### 🚀 Enhancements and New Features
+
+- Add metalad extractor using `singularity inspect`.
```

Comment:
not liking name 100% but due to singlularity/apptainer duality let's keep as is and do mention in changelog
```suggestion
- Add metalad extractor `container_inspect` using `singularity` or `apptainer` `inspect`.
```